pokemongo_bot/cell_workers/incubate_eggs.py

```diff
@@ -38,7 +38,7 @@ def work(self):
                     data={
                         'km_needed': self.used_incubators[0]['km_needed'],
                         'distance_in_km': km_left,
-                        'eggs': len(self.eggs),
+                        'eggs': len(self.eggs) + len(self.used_incubators),
```

Comment:
It seems like since we are already sending the length of the used incubators, we want to keep the number of eggs separate from the number of incubated eggs.